orchagent/muxorch.cpp

```diff
@@ -1927,6 +2089,106 @@ void MuxOrch::updateCachedNeighbors()
     }
 }
 
+void MuxOrch::saveMuxNeighbors()
```

Comment:
I'm not seeing this called anywhere, Is it currently used? If not, can you remove from this PR (as well as any supporting code) and it can be re-evaulated at a later point w/warmboot.